high_card.rb

```diff
@@ -1,5 +1,44 @@
+require 'pry'
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
-
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "Jack", "Queen", "King", "Ace"]
+suits = [ "Hearts", "Spades", "Clubs", "Diamonds" ]
+deck = []
+player_input = ""
 players = []
+
+# Creates Deck with 52 cards and index values for comparing
+ranks.each_with_index do |rank, i|
+  suits.each do |suit|
+    deck << [i + 2,rank,suit]
```

Comment:
I don't think you need to add 2 to each index, you should be able to compare them without adding any additional numbers